autogen/agentchat/groupchat.py

```diff
@@ -633,12 +645,17 @@ def validate_speaker_name(recipient, messages, sender, config) -> Tuple[bool, Un
             human_input_mode="NEVER",  # Suppresses some extra terminal outputs, outputs will be handled by select_speaker_auto_verbose
         )
 
+        # Add the message transforms, if any, to the speaker selection agent
+        if self._auto_message_transforms is not None:
+            self._auto_message_transforms.add_to_agent(speaker_selection_agent)
```

Comment:
What do you think about calling `_auto_message_transforms` `_speaker_selection_transforms`? I found it confusing to follow along

Reply:
Good suggestion, looking back on it, it isn't very clear - I've changed it and will commit shortly.